lua/entities/gmod_wire_cpu.lua

```diff
@@ -244,7 +244,7 @@ WireLib.AddInputAlias( "NMI", "Interrupt" )
 
 function ENT:TriggerInput(iname, value)
 	if iname == "Clk" then
-		self.Clk = value
+		self.Clk = (value ~= 0 and 1 or 0)
```

Comment:
Later code checks for this being `== 1`, deeper in ZVM.

Is there a standard convention for boolean inputs? Both `~= 0` and `>= 1` seem to make equal sense to me, so I guess I should change one of them for consistency.